quic/s2n-quic-core/src/havoc.rs

```diff
@@ -354,6 +391,46 @@ impl Strategy for Disabled {
     fn havoc<R: Random>(&mut self, _rand: &mut R, _buffer: &mut EncoderBuffer) {}
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct Hold<S: Strategy> {
+    strategy: S,
+    min: u64,
+    max: u64,
+    value: Vec<u8>,
```

Comment:
We'll probably need to conditionally compile this strategy based on `feature = "std"`.

Reply:
Or I guess `feature = "alloc"` is all you need 😁